pkgs/top-level/python-packages.nix

```diff
@@ -22212,6 +22212,25 @@ in modules // {
     };
   };
 
+  yolk = buildPythonPackage rec {
```

Comment:
If this is an application (command line tool), `buildPythonApplication` should be preferred over `buildPythonPackage` and it should be placed independently in the pkgs tree (not in `python-packages.nix` which is used for packages applications rely on).

`git grep buildPythonApplication` should give you many examples.

If you want, there is a pending documentation PR (#13320) that include discussion about this subject.